scripts/bash/enceladus_env.template.sh

```diff
@@ -120,3 +120,10 @@ ADDITIONAL_JVM_EXECUTOR_CONF_CLUSTER="$KRB5_CONF_CLUSTER $TRUST_STORE_CLUSTER $T
 # Switch that tells the script if it should exit if it encounters unrecognized.
 # On true it prints an Error and exits with 127, on false it only prints a warning
 EXIT_ON_UNRECOGNIZED_OPTIONS="true"
+
+# Variables for the s3a wrapper implementation
+MENAS_API="http://localhost:8080/menas/api"
+ECS_API_BASE="https://localhost"
+ECS_API_KK="$ecs_api_base/kk"
+ECS_API_MAP="$ecs_api_base/map"
```

Comment:
```suggestion
ECS_API_KK="$ecs_api_base/kk"
```
Why having both KK and MAP endpoint when the MAP one does not seem to be used?

Reply:
Waiting for further instructions on doing an automatic jceks file resolution. 

TODO in code - https://github.com/AbsaOSS/enceladus/pull/2198/files#diff-82709faf8052f83cfd18429a9f6451da34687d8261923b52d065464293c11c14R24

The map endpoint was suggested, but it does not work both ways. I will remove depending on the on next steps.